content/docs/faq-versioning.md

```diff
@@ -1,64 +1,64 @@
+Sebaliknya, kami merilis fitur-fitur baru dalam versi-versi kecil. Yang artinya rilis-rilis kecil tersebut seringkali lebih menarik dan lebih memikat dibandingkan perubahan-perubahan besar, meskipun namanya sederhana.
 
-### Commitment to Stability {#commitment-to-stability}
+### Komitmen terhadap Stabilitas {#commitment-to-stability}
 
-As we change React over time, we try to minimize the effort required to take advantage of new features. When possible, we'll keep an older API working, even if that means putting it in a separate package. For example, [mixins have been discouraged for years](/blog/2016/07/13/mixins-considered-harmful.html) but they're supported to this day [via create-react-class](/docs/react-without-es6.html#mixins) and many codebases continue to use them in stable, legacy code.
+Karena kami mengubah React setiap waktu, kami mengupayakan untuk meminimalisasi usaha yang dibutuhkan untuk memanfaatkan fitur-fitur baru. Ketika memungkinkan, kami akan tetap menjaga agar API yang lama tetap berfungsi, bahkan jika itu berarti menaruhnya dalam *package* terpisah. Misalnya, [*mixins* sudah tidak disarankan selama bertahun-tahun](/blog/2016/07/13/mixins-considered-harmful.html) tetapi masih didukung hingga sekarang [melalui *create-react-class*](/docs/react-without-es6.html#mixins) dan banyak basis kode yang tetap menggunakannya dalam kode *legacy* yang stabil.
 
-Over a million developers use React, collectively maintaining millions of components. The Facebook codebase alone has over 50,000 React components. That means we need to make it as easy as possible to upgrade to new versions of React; if we make large changes without a migration path, people will be stuck on old versions. We test these upgrade paths on Facebook itself – if our team of less than 10 people can update 50,000+ components alone, we hope the upgrade will be manageable for anyone using React. In many cases, we write [automated scripts](https://github.com/reactjs/react-codemod) to upgrade component syntax, which we then include in the open-source release for everyone to use.
+Lebih dari satu juta pengembang menggunakan React, secara kolektif merawat jutaan komponen. Basis kode Facebook sendiri mempunyai lebih dari 50,000 komponen. Yang artinya kami harus membuatnya semudah mungkin untuk meng-*upgrade* versi React yang baru; jika kami membuat perubahan-perubahan besar tanpa jalur migrasi, banyak orang akan terjebak pada versi yang lama. Kami menguji jalur-jalur pembaruan ini di Facebook sendiri - jika tim kami yang kurang dari 10 orang dapat memperbarui 50,000+ komponen sendiri, kami berharap pembaruannya dapat dikelola oleh semua orang yang menggunakan React. Dalam beberapa kasus, kami menulis [skrip otomatis](https://github.com/reactjs/react-codemod) untuk memperbarui sintaks komponen, yang kemudian kami sertakan dalam rilis *open-source* untuk digunakan semua orang.
```

Comment:
what is the reason to not translate _meng-*upgrade*_ into "memperbarui" ?

Reply:
Sorry, my bad, I translate it to "memperbarui" at first, but after I look for the word in Glosarium I didn't find it and in the end I told myself "let's try this 'meng-upgrade' word first".

Reply:
@asharimh97 Let's use "memperbarui" instead. 🙏